### DISCUSSION TITLE
スタンプが表示されないです！

### DISCUSSION THREAD

**Discussion by KOTARO1209** [lit-iphone-course/lit-iphone-course#138] [Question-original] [answered]
## (1) 実現したいこと
スタンプを表示したい
## (2) 実現するために自分が試した内容とその結果
教科書を31ページまでやりました
## (3) 調べた内容や資料
教科書：スタンプ
## (4) 原因だと思われそうな箇所(ソースコードをコピペして貼る)
実行はできるんですけど、スタンプが表示されない状態です。原因がわからないです🙇‍♂️
## (5) GitHubのリンク
https://github.com/KOTARO1209/Stamp

**Comment by appleEmily** [comment, marked as answer]
<img width="396" alt="image" src="https://user-images.githubusercontent.com/49332380/219056207-f43a1798-c0ca-441b-b51f-bb33a8ffb0ab.png">
ちゃんとできてたよー。

アプリの仕様としては、下の画像でどのスタンプにするか選ぶ→上のフィールドタップするとスタンプ押せるって感じ

**Reply by KOTARO1209** [reply]
なるほど！！ありがとうございます🙇‍♂️
